### ISSUE TITLE
uses outdated version of squashfs-tools

### ISSUE THREAD

**Issue by zgaleaz** [devttys0/sasquatch#49]
sasquatch is currently using an old version of squashfs-tools, which contains the bug described here: https://github.com/plougher/squashfs-tools/issues/125 that was fixed in a later version.

https://github.com/ReFirmLabs/binwalk depends on this package, and after it (silently) failed to extract a squashfs binary while running in a docker container with an almost infinite open file limit, I tracked the upstream issue to here.

In summary: just bump the squashfs version in the build script
